docs/modules/k6.md

```diff
@@ -40,6 +42,16 @@ func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomize
 
 When starting the K6 container, you can pass options in a variadic way to configure it.
 
+#### Image
+
+!!! warning
+    The K6 module uses a custom Docker image to build the `k6` binary with all the required extensions. This image is based on the official `k6` image and it is available on [Docker Hub](https://hub.docker.com/r/szkiba/k6x). Therefore, only the `szkiba/k6x` image should be used with this module.
```

Comment:
I find this warning repetitive as this is explained in the description of the module. Also, there are some inaccuracies in the description of k6x and k6 in the context of this module. I would be more concise: 

```suggestion
    The K6 module uses a `k6x` image to build a `k6` binary with all the required extensions. Therefore, only the [szkiba/k6x](https://hub.docker.com/r/szkiba/k6x) image should be used with this module.
```

Reply:
Do we want to remove the entire warning. I'm fine doing it if you consider it redundant

Reply:
I think a short warning here is convenient.